tests_integration/test_warehouse.py

```diff
@@ -7,7 +7,7 @@
 @pytest.mark.integration
 @mock.patch("snowcli.cli.warehouse.print_db_cursor")
 def test_warehouse_status_query(mock_print, runner, snowflake_session):
-    runner.invoke_with_config(["warehouse", "status"])
+    runner.invoke_with_config_and_integration_connection(["warehouse", "status"])
```

Comment:
Maybe we should introduce `integration_runner` to avoid mixing responsibilities in single fixture?

Reply:
I'm not sure if it resolves any problem. We have two ways of execution commands in integration tests (with and without `-c integration` flag) but both are correct and both are used by integration tests.

Adding another "runner" fixture will make you think which runner to use instead of which method to use. I don't see any benefit here, only complications.